Packs/AWS-Athena/Integrations/AWS-Athena/AWS-Athena.py

```diff
@@ -44,205 +19,180 @@ def default(self, obj):
-            aws_access_key_id=sts_response['Credentials']['AccessKeyId'],
-            aws_secret_access_key=sts_response['Credentials']['SecretAccessKey'],
-            aws_session_token=sts_response['Credentials']['SessionToken'],
-            verify=VERIFY_CERTIFICATE,
-            config=config
-        )
-    else:
-        if region is not None:
-            client = boto3.client(
-                service_name=service,
-                region_name=region,
-                aws_access_key_id=AWS_ACCESS_KEY_ID,
-                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
-                verify=VERIFY_CERTIFICATE,
-                config=config
-            )
-        else:
-            client = boto3.client(
-                service_name=service,
-                region_name=AWS_DEFAULT_REGION,
-                aws_access_key_id=AWS_ACCESS_KEY_ID,
-                aws_secret_access_key=AWS_SECRET_ACCESS_KEY,
-                verify=VERIFY_CERTIFICATE,
-                config=config
-            )
-
-    return client
-
-
-def start_query_execution_command(args):
-    client = aws_session(
-        region=args.get('region'),
-        roleArn=args.get('roleArn'),
-        roleSessionName=args.get('roleSessionName'),
-        roleSessionDuration=args.get('roleSessionDuration'),
-    )
-    data = []
-    kwargs = {'QueryString': args.get('QueryString')}
-    if args.get('ClientRequestToken') is not None:
-        kwargs.update({'ClientRequestToken': args.get('ClientRequestToken')})
-    if args.get('Database') is not None:
-        kwargs.update({'QueryExecutionContext': {'Database': args.get('Database')}})
-    if args.get('OutputLocation') is not None:
-        kwargs.update({'ResultConfiguration': {'OutputLocation': args.get('OutputLocation')}})
-    if args.get('EncryptionOption') is not None:
-        kwargs.update({'ResultConfiguration': {'EncryptionConfiguration': {'EncryptionOption': args.get('EncryptionOption')}}})
-    if args.get('KmsKey') is not None:
-        kwargs.update({'ResultConfiguration': {'EncryptionConfiguration': {'KmsKey': args.get('KmsKey')}}})
-    if args.get('WorkGroup') is not None:
-        kwargs.update({'WorkGroup': args.get('WorkGroup')})
+def parse_rows_response(rows_data: list[dict]) -> list[dict]:
+    """
+    Parse and arrange the 'Rows' data from the response.
+
+    Args:
+        rows_data (list[dict]): The 'Rows' data from the response.
+
+    Returns:
+        list[dict]: The data in a parsed and arranged format.
+    """
+    keys: list[str] = [item['VarCharValue'] for item in rows_data[0]['Data']]
+    raw_results = [item['Data'] for item in rows_data[1:]]
+    result_data = []
+
+    for raw_result in raw_results:
+        current_item_data = {}
+
+        for idx, value in enumerate(raw_result):
+            if 'VarCharValue' in value:
+                current_item_data[keys[idx]] = value['VarCharValue']
+
+        result_data.append(current_item_data)
+
+    return result_data
```

Comment:
Example for such a response can be seen on:
https://github.com/demisto/content/pull/30694/files#diff-40ceff67b244e3d04a82f5a5018a9d8781b6bcf2d8c230f632757af55a5eca09
(first item is keys, and then after that, each item is the values for these keys in corresponding indexes. Like a CSV table).